### PR TITLE
Fix for issue #30 in upstream:

### PR DESCRIPTION
**Pull request by jeffstoner**
We are enhancing the File System block of the System Info admin section to give more context to the user as to what the object is and, in the case of files, the last modification time and size.

### CODE REVIEW

File: bonfire/Language/en/Bonfire.php [hunk 1]
```diff
@@ -14,4 +14,11 @@
 
     'notAuthorized' => 'You do not have the correct permissions.',
     'unknownAction' => 'Did not recognize the action you requested.',
+
+    'nested' => [
+        'fileSystem' => [
+            'lastModified' => 'Last Modified',
```
Comment (lonnieezell) [inline bonfire/Language/en/Bonfire.php]: No need to make these this specific. They're pretty generic phrases that can potentially be reused elsewhere. Please make them top-level standalone entries like the rest of them.

**Review by lonnieezell — CHANGES_REQUESTED** [review]
It looks great. Make the language file a bit less specific and we're good. Thanks!

File: bonfire/Language/en/Bonfire.php (hunk 1 above)
Comment (jeffstoner) [reply bonfire/Language/en/Bonfire.php]: Done.

**Review by lonnieezell — APPROVED** [review]

**Comment by lonnieezell** [comment]
Looks great. Thanks!

**Pull request merged**